store/src/java/com/zimbra/cs/imap/ImapDaemon.java

```diff
@@ -119,6 +120,8 @@ public static void main(String[] args) {
                     errorExit("ImapDaemon: imapd service was unable to intialize timezones EXITING.");
                 }
 
+                ExtensionUtil.init("com.zimbra.cs.network.license.LicenseExtension");
```

Comment:
Would be good to have a comment saying that this loads the LicenseExtension if it is present and that this is used to allow users to be notified of licensing issues rather than just getting unexpected behavior.

Talking of which, what happens on FOSS systems?  The comment should also re-assure that this
What testing has been done? with this change, do users get a warning if their license has expired?

Reply:
Yes the alert is now returned when you open a connection to the server.
In the case of FOSS it will return a 'WARN' level log message in the imapd.log file but has no other impact.

I'll add the comment.